src/nvim/ex_cmds.lua

```diff
@@ -2678,18 +2678,18 @@ return {
     func='ex_ni',
   },
   {
-    command='tearoff',
-    flags=bit.bor(NEEDARG, EXTRA, TRLBAR, NOTRLCOM, CMDWIN),
-    addr_type=ADDR_LINES,
-    func='ex_tearoff',
-  },
-  {
     command='terminal',
     flags=bit.bor(BANG, FILES, CMDWIN),
     addr_type=ADDR_LINES,
     func='ex_terminal',
   },
   {
+    command='tearoff',
+    flags=bit.bor(NEEDARG, EXTRA, TRLBAR, NOTRLCOM, CMDWIN),
+    addr_type=ADDR_LINES,
+    func='ex_tearoff',
+  },
```

Comment:
> This array must be kept in alphabetical order.

It's not currently in strict alphabetical order. For example, `cgetbuffer` appears after `cgetfile`. It appears that commands are regularly reordered for this purpose (for example, see https://github.com/neovim/neovim/pull/2927#issuecomment-117628390 regarding the new `:cfdo` command).